specification/containerservice/resource-manager/Microsoft.ContainerService/preview/2018-09-30-preview/examples/OpenShiftManagedClustersGet.json

```diff
@@ -60,7 +60,8 @@
                 "provider": {
                   "kind": "AADIdentityProvider",
                   "clientId": "clientId",
-                  "tenantId": "tenantId"
+                  "tenantId": "tenantId",
+                  "customerAdminGroupId": "customerAdminGroupId"
```

Comment:
>customerAdminGroupId [](start = 43, length = 20)

This is kind of misleading since the value is a GUID. The way this is typically represented in the Swagger is by either enclosing the content in curly braces ("{customerAdminGroupId}", for example) or by just listing empty GUID in the format your service returns ("00000000-0000-0000-0000-000000000000", for example). I have the same comment for all the AAD-related properties in all the existing examples.

Reply:
updated